src/renderer/containers/Table/CustomCells/ProgramCell/index.tsx

```diff
@@ -1,34 +1,35 @@
 import { Select } from "antd";
 import { castArray } from "lodash";
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
 import { CellProps } from "react-table";
 
-import { AnnotationName } from "../../../../constants";
-import { getAnnotations, getAnnotationOptions } from "../../../../state/metadata/selectors";
+import { AnnotationOption } from "../../../../services/labkey-client/types";
+import { requestProgramOptions } from "../../../../state/metadata/actions";
+import { getProgramOptions } from "../../../../state/metadata/selectors";
 import { FileModel } from "../../../../state/types";
 import { updateUpload } from "../../../../state/upload/actions";
 import DisplayCell from "../../DefaultCells/DisplayCell";
 
+
 const { Option } = Select;
 
 export default function ProgramCell(props: CellProps<FileModel, string[]>) {
   const dispatch = useDispatch();
 
-  const annotations = useSelector(getAnnotations);
-  const options = useSelector(getAnnotationOptions);
-
-  // Get options for program annotation
-  const programAnnotation = annotations.find(a => a.name === AnnotationName.PROGRAM);
-  const programOptions = programAnnotation
-    ? options.filter(opt => opt.annotationId === programAnnotation.annotationId)
-    : [];
+  const programOptions = useSelector(getProgramOptions);
```

Comment:
Looks like the logic for setting program options moved to a different Redux layer?